test/test_main.py

```diff
@@ -0,0 +1,137 @@
+# -*- coding: utf-8 -*-
+from binascii import unhexlify
+from pathlib import Path
+from sys import version_info
+from warnings import filterwarnings
+
+import pytest
+
+from puremagic.main import what
+
+filterwarnings("ignore", message="'imghdr' is deprecated")
+try:  # imghdr was removed from the standard library in Python 3.13
+    from imghdr import what as imghdr_what
+except ModuleNotFoundError:
+    imghdr_what = None  # type: ignore[assignment]
+
+# file_tests = sorted(test_func.__name__[5:] for test_func in imghdr.tests)
+# file_tests = "bmp exr gif jpg pbm pgm png ppm ras rgb tif webp xbm".split()
+file_tests = "bmp gif jpg png tif webp".split()  # TODO: (cclauss) Add the remaining files
```

Comment:
```suggestion
file_tests = ["bmp", "gif", "jpg", "png", "tif", "webp"]
```
(And add any missing)